src/Microsoft.TestPlatform.PlatformAbstractions/uap10.0/Tracing/PlatformEqtTrace.cs

```diff
@@ -64,7 +64,17 @@ public void WriteLine(PlatformTraceLevel level, string message)
         /// <inheritdoc/>
         public bool InitializeVerboseTrace(string customLogFile)
         {
-            LogFile = Path.GetTempPath() + Path.GetFileNameWithoutExtension(customLogFile).Replace(" ", "_") + ".TpTrace.log";
+            string logFileName = string.Empty;
+            try
+            {
+                logFileName = Path.GetFileNameWithoutExtension(customLogFile.TrimStart('"').TrimEnd('"')).Replace(" ", "_");
+            }
+            catch
+            {
+                logFileName = Guid.NewGuid().ToString();
+            }
+
+            LogFile = Path.GetTempPath() + logFileName + ".TpTrace.log";
```

Comment:
Should we use `Path.Conbine` here?

Reply:
it's all same internally

Reply:
`Path.GetTempPath()` value depends on [Env variable](https://msdn.microsoft.com/en-us/library/system.io.path.gettemppath(v=vs.110).aspx). User may set it to C:\Temp or C:\Temp.

Reply:
the behavior for UWP is different, all BCL have implementation according to CLR, UWP apps cannot write in C:\Temp, they are allocated their own temp folder inside app local data.

Reply:
C:\Temp was just an example I gave. In UWP how the Path.GetTempPath() get determined? User can modify that? If so can he ignore `\` at end?

Reply:
no idea how it is overwritten.

Reply:
Then we better use `Path.Combine()`.

Reply:
lets use Path.Combine. There is a reason why Path.Combine is there in BCL and we shouldn't re implement it